### PR TITLE
add support for ebcdic 278/1143 code pages

### PR DESCRIPTION
**Pull request by zmyrgel**
add more ebcdic code pages

### CODE REVIEW

File: src/enc-ebcdic.lisp [hunk 1]
```diff
@@ -30,6 +30,14 @@
     "An alleged character set used on IBM dinosaurs."
   :aliases '(:ibm-037))
 
+(define-character-encoding :ebcdic-278
+    "An character set used on IBM mainframes in Finland/Sweden."
```
Comment (luismbo) [inline src/enc-ebcdic.lisp]: s/An/A

File: src/enc-ebcdic.lisp [hunk 2]
```diff
@@ -30,6 +30,14 @@
...
+  :aliases '(:ibm-278))
+
+(define-character-encoding :ebcdic-1143
+    "An character set used on IBM mainframes in Finland/Sweden using Euro sign."
```
Comment (luismbo) [inline src/enc-ebcdic.lisp]: s/An/A

**Comment by zmyrgel** [comment]
While I'm adding tests, could there be way to handle newlines in EDCDIC?

EBCDIC uses next-line character to define new lines so after conversion the text is in single long lisp string.
Could there be some optional parameter in the which would tell if the next-line character would be mapped to newline during conversion?

**Comment by zmyrgel** [comment]
Would this be enough?

**Pull request merged**